docs/user-guide/building-assistants.rst

```diff
@@ -903,6 +1012,28 @@ let’s say the user asks "I want to apply for a job at Rasa", we can then reply
    * ask_job
      utter_job_not_handled
 
+.. note::
+    Here's a minimal checklist of files we modified to help our assistant fail gracefully:
+
+      - ``data/nlu.md``: 
+          - add training data for the ``out_of_scope`` intent
+          - add training data for any specific out of scope intents that you want to handle seperately
+      - ``data/stories.md``:
+          - add stories for any specific out of scope intents that you want to handle seperately
+      - ``domain.yml``: 
+          - add the ``out_of_scope`` intent
+          - add an ``utter_out_of_scope`` response
+          - add any specific out of scope intents that you want to handle seperately
+          - add responses for any specific out of scope intents that you want to handle seperately
+      - ``actions.py``: 
+          - customise ``ActionDefaultAskAffirmation`` to suggest intents for the user to choose from
+      - ``config.yml``: 
+          - add the TwoStageFallbackPolicy to the ``policies`` section
+          - define the ``nlu_threshold`` for the TwoStageFallbackPolicy
```

Comment:
can you make sure these checklists are consistently capitalised? Some are lowercase at the start, others uppercase. I already normalised it for the first two.

Reply:
also, is the nlu_threshold step necessary? Do we actually modify it from the default?

Reply:
nope